### PR TITLE
swf: Bail out if swf version is 0

### PR DESCRIPTION
**Pull request by Toad06**
[Sample](https://github.com/ruffle-rs/ruffle/files/11468224/swfv0.zip) (hex-edited)

### CODE REVIEW

File: swf/src/read.rs [hunk 1]
```diff
@@ -73,6 +73,16 @@ pub fn decompress_swf<'a, R: Read + 'a>(mut input: R) -> Result<SwfBuf> {
     let version = input.read_u8()?;
     let uncompressed_len = input.read_u32::<LittleEndian>()?;
 
+    // Check whether the SWF version is 0.
+    // Note that the behavior should actually vary, depending on the player version:
+    // - Flash Player 9 and earlier bail out (the behavior we implement).
```
Comment (torokati44) [inline swf/src/read.rs:78]: Typo: "and later"?

Comment (Toad06) [reply swf/src/read.rs]: Whoops! Thanks for noticing this. 😉

**Comment by Dinnerbone** [comment]
Neat, thanks! Is there any content that did this or is this just in case? :D

**Review by Dinnerbone — APPROVED** [review]

**Comment by Toad06** [comment]
Yeah, I really tested that out of curiosity.

And thank you for the review. 😉

**Pull request merged**